src/openms/include/OpenMS/KERNEL/MSChromatogram.h

```diff
@@ -51,30 +51,26 @@ namespace OpenMS
     @brief The representation of a chromatogram.
     @ingroup Kernel
   */
-  template <typename PeakT>
-  class MSChromatogram :
-    private std::vector<PeakT>,
+
+  class OPENMS_DLLAPI MSChromatogram :
```

Comment:
I agree as only complicated functions are moved to the cpp. But lets not move simple functions there as this will incur an overhead of a function call without LTO